ledger-core/v2/logicrunner/sm_test_api/sm.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+package sm_test_api
+
+import (
+	"github.com/insolar/assured-ledger/ledger-core/v2/conveyor"
+	"github.com/insolar/assured-ledger/ledger-core/v2/conveyor/smachine"
+	"github.com/insolar/assured-ledger/ledger-core/v2/insolar"
+	"github.com/insolar/assured-ledger/ledger-core/v2/insolar/gen"
+	"github.com/insolar/assured-ledger/ledger-core/v2/insolar/payload"
+	"github.com/insolar/assured-ledger/ledger-core/v2/logicrunner/s_sender"
+	"github.com/insolar/assured-ledger/ledger-core/v2/reference"
+	"github.com/insolar/assured-ledger/ledger-core/v2/vanilla/injector"
+	"github.com/insolar/assured-ledger/ledger-core/v2/vanilla/throw"
+)
+
+type TestApiCallSM struct {
+	requestPayload  payload.VCallRequest
+	responsePayload payload.VCallResult
+
+	response chan payload.VCallResult
+
+	// injected arguments
+	PulseSlot *conveyor.PulseSlot
+	sender    *s_sender.SenderServiceAdapter
+}
+
+type TestApiCallSMDeclaration struct {
+	smachine.StateMachineDeclTemplate
+}
+
+var ApiCaller, _ = insolar.NewObjectReferenceFromString("insolar:0AAABAnRB0CKuqXTeTfQNTolmyixqQGMJz5sVvW81Dng")
```

Comment:
APICaller won't be NodeID? Or we have no NodeID right now?

Reply:
I think right now it doesn't matter, can we win something to change it to NodeID?